### ISSUE TITLE
Implement property changes for UNI and NORM distributions

### ISSUE THREAD

**Issue by sebjameswml** [SpineML/SpineML_PreFlight#3]
So user could call something like:

```
spineml_preflight -e /path/to/expt.xml --property_change="Population1:a:UNI(1,2,123)"
```

**Comment by sebjameswml** [comment]
NORM(2.6,1.1,124) would expand to:
```xml
<UL:Property alias="SPF" name="S" dimension="?">
  <UL:NormalDistribution mean="2.6" variance="1.1" seed="124"/>
</UL:Property>
```

UNI(1,2,124) would expand to:
```xml
<UL:Property alias="SPF" name="S" dimension="?">
  <UL:UniformDistribution minimum="1" maximum="2" seed="124"/>
</UL:Property>
```

**Comment by sebjameswml** [comment]
Fixed with commit 5f5d632
